README.md

```diff
@@ -145,3 +145,13 @@ Contributions are welcome & greatly appreciated, every little bit helps. Make co
 ```
 make test-docker
 ```
+
+### Interactive "Office Hours"
+
+The maintenance team plans to be available regularly for questions, comments,
+pings, etc for about an hour twice a week. The current schedule is:
+
+* 2:00pm US/Eastern (currently 18:00 UTC) Mondays
+* 9:00am US/Eastern (currently 13:00 UTC) Thursdays
```

Comment:
I have no relation with US/Eastern, and I do not seem to get it to work like `date -d '2:00 pm UTC'`. Care to add a command to convert the time to a local one? Or maybe point to https://www.timeanddate.com/worldclock/usa/boston or the like.

Other than this, it looks good to me.

Once merged, add an event to the [Ceph Community calendar](https://calendar.google.com/calendar/embed?src=9ts9c7lt7u1vic2ijvvqqlfpo0%40group.calendar.google.com) as well, please.

Reply:
Man, I really hate dealing with time. :-)
I changed it after verifying `date` works with the new version.